corehq/apps/fixtures/fixturegenerators.py

```diff
@@ -61,11 +60,29 @@ def __call__(self, restore_state):
                 global_types[data_type._id] = data_type
             else:
                 user_types[data_type._id] = data_type
-        items = []
+        items = global_items = user_items = []
```

Comment:
Bad bad bad. Very bad. Glad I put this on staging.

Reply:
OK I over-racted. This was a bug, but it only affected metrics. Below global_items and user_items are reassigned if they're not empty, and otherwise they're not used except for metrics. That's why tests didn't catch this.